### ISSUE TITLE
Frozen node report of some values on 8.23.1

### ISSUE THREAD

**Issue by james-1987** [zwave-js/zwave-js-ui#3243]
### Checklist

- [X]  I am not using Home Assistant. Or: a developer has told me to come here.
- [X] I have checked the troubleshooting section and my problem is not described there.
- [X] I have read the changelog and my problem is not mentioned there.

### Deploy method

Docker

### Z-Wave JS UI version

8.23.1

### ZwaveJS version

11.13.0

### Describe the bug

My 2 Heatit Z-TRM3 termostat freezes node report of temperatur value: [5-49-2(3 and 4)-Air temperature]. When i updated to 8.23.1 from 8.22.3. Other value like [5-50-1-value-66561] Electric Consumption [V] worked. You could turn the termostat relay on and off.

I tried:
Heal Node
Refresh Values
Re-interview Node
Clear Retained
Update topics
Took of fuse to termostat for a hard reboot.

My setup: 
AEON Labs Z‐Stick Gen5 USB Controller ZW090 FW: v1.2 SDK: v6.81.6 
Heatit Floor thermostat Z-TRM3 FW: v4.0.33 SDK: v6.81.6

I got back to 8.22.3 again and everything started report again. I do not now if this problem is in  8.23.0 because i don't install "XX.0" versions.

I do not want to go back to again to 8.23.1 to collect info for bug report because z-wave network is in a critical place that it need to work. (company)

### To Reproduce

Having 8.23.1 install.

### Expected behavior

Not have frozen values.

**Comment by toffyrn** [comment]
I experienced the same behaviour on several heatit thermostats when upgrading zwave js UI from 1.15.6 to 1.15.7 in home assistant. I believe that is the same as upgrading from 8.23.0 to 8.23.1.

If needed, I don't mind upgrading again and do some debugging. Just let me know what information is needed.

**Comment by ScratMan** [comment]
I also had such issue on a FIBARO FGK101 with a DSB18B20 sensor. The battery died, and after replacing the external temperature sensor value went down to -1800°C 👀 
I tried several time to wake up the device, I managed to get a correct temperature readout, but it was not updating after. So I decided to exclude the device and included it again, but the temperature sensor never came back.

I had to go back to 8.22 to be able to include the device and get the temperature sensor back.
